Specs/Sequence/Korean/PhoneNumberModel.json

```diff
@@ -1,7 +1,7 @@
 [
   {
     "Input": "Tel: (+31)12-3456789.",
-    "NotSupported":"javascript",
+    "NotSupported":"javascript,java",
```

Comment:
Why is this not supported? Can you add comments here or in the PR description for the reason to disable all these specs?

Reply:
There is no reason to be `notsupported`, we updated the list of specs 😊